### PR TITLE
Add Event Hubs and Service Bus release notes.

### PR DESCRIPTION
**Pull request by conniey**
* Add Event Hubs and Service Bus release notes.

### CODE REVIEW

File: releases/2020-09/java.md [hunk 1]
```diff
@@ -24,6 +26,7 @@ The Azure SDK team is pleased to announce our September 2020 client library rele
 - Management Library - Network
 - Management Library - Resources
 - Management Library - Storage
+- Azure Service Bus
```
Comment (samvaity) [inline releases/2020-09/java.md:29]: Service bus still just preview, should be removed from here

File: releases/2020-09/java.md [hunk 2]
```diff
@@ -10,8 +10,10 @@ The Azure SDK team is pleased to announce our September 2020 client library rele
 
 #### GA
 
+- Azure Core AMQP
 - Azure Core Serializer GSON JSON
 - Azure Core Serializer Jackson JSON
+- Azure Event Hubs
```
Comment (samvaity) [inline releases/2020-09/java.md:16]: Eventhubs entry can go under Updates

File: releases/2020-09/java.md [hunk 3]
```diff
@@ -136,6 +140,18 @@ To use the GA and beta libraries, refer to the Maven dependency information belo
...
+    <artifactId>azure-messaging-eventhubs</artifactId>
+    <version>5.2.0</version>
+</dependency>
+
+<dependency>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-messaging-servicebus</artifactId>
+    <version>7.0.0-beta.6</version>
+</dependency>
+
```
Comment (samvaity) [inline releases/2020-09/java.md:154]: Missing entry for Amqp here and changelog

File: releases/2020-09/java.md (hunk 1 above)
Comment (conniey) [reply releases/2020-09/java.md:29]: Oh shoot

File: releases/2020-09/java.md [hunk 4]
```diff
@@ -10,6 +10,7 @@ The Azure SDK team is pleased to announce our September 2020 client library rele
 
 #### GA
 
+- Azure Core AMQP
```
Comment (samvaity) [inline releases/2020-09/java.md:13]: AMQP needs to go in updates section

**Review by samvaity — APPROVED** [review]

File: releases/2020-09/java.md (hunk 4 above)
Comment (conniey) [reply releases/2020-09/java.md:13]: Oh junk

**Pull request merged**